geode-core/src/distributedTest/java/org/apache/geode/cache30/PutAllMultiVmDUnitTest.java

```diff
@@ -486,24 +443,24 @@ public void run2() throws CacheException {
           // do nothing
```

Comment:
On line 441 above this, rather than using `fail()` when we expect a call to raise a throwable but it doesn't, it would be better to use `assertThatThrownBy()` to assert that we see the expected exception. This provides a more descriptive failure in the event that we don't see the expected exception. This comment also applies to the other 4 uses of `fail()` in this test.

Reply:
Nice. Very clean.